deepaugment/augmenter.py

```diff
@@ -0,0 +1,148 @@
+# (C) 2019 Baris Ozmen <hbaristr@gmail.com>
+
+import numpy as np
+from imgaug import augmenters as iaa
+
+
+def normalize(X):
+    return (X/255.).copy()
```

Comment:
You use 255 a lot in the code. Is that something you want to set as a parameter in your config file as well?